app/backend/approaches/approach.py

```diff
@@ -190,8 +191,9 @@ async def compute_image_embedding(self, q: str, vision_endpoint: str, vision_key
         data = {"text": q}
 
         async with aiohttp.ClientSession() as session:
-            async with session.post(url=endpoint, params=params, headers=headers, json=data) as response:
-                response.raise_for_status()
+            async with session.post(
```

Comment:
The raise_for_status option appears to achieve the same as calling raise_for_status, and it's easier on our mocks. :)